src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -241,6 +243,38 @@ public async Task<IActionResult> PerformerDelete(PerformerListViewModel model)
                 new { page = model.PaginateModel.CurrentPage });
         }
 
+        [HttpPost]
+        public async Task<IActionResult> PerformerSelectionDelete
+            (PerformerSelectionsViewModel model)
+        {
+            var settings = await _performerSchedulingService.GetSettingsAsync();
+            var schedulingStage = _performerSchedulingService.GetSchedulingStage(settings);
+            if (schedulingStage < PsSchedulingStage.SchedulingOpen)
+            {
+                return RedirectToAction(nameof(PerformerManagementController.PerformerSelections),
+                    new { id = model.Performer.Id });
+            }
+            try
+            {
+                var branchSelection = await _performerSchedulingService
+                    .GetBranchProgramSelectionByIdAsync(model.BranchSelectionId);
+                await _performerSchedulingService.DeleteBranchSelectionAsync(branchSelection);
+                var branch = await _siteService.GetBranchByIdAsync(branchSelection.BranchId);
+                var program = await _performerSchedulingService
+                    .GetProgramByIdAsync(branchSelection.ProgramId.Value);
+                branchSelection.IsDeleted = true;
```

Comment:
Can be removed, IsDeleted is set in DeleteBranchSelectionAsync()

Reply:
Fixed in 9bf3f38be7dda584a234205340b36076a97d5cc0